client/src/Header.js

```diff
@@ -0,0 +1,47 @@
+import './App.css';
+import {useState} from 'react'
+import {BsFillCartFill} from 'react-icons/bs'
+import { BsFillSuitHeartFill } from 'react-icons/bs'
+import { ImSearch } from 'react-icons/im'
+
+export default function Header() {
+  const [search, setSearch] = useState('')
+
+  const handleChange = (event) => {
+    setSearch(event.target.value);
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await fetch(`https://www.googleapis.com/books/v1/volumes?q=${search}&key=AIzaSyCQ7EBy8yMIYuOdh350WfDgEwWKGJSWAh4`)
+      const jsonData= await response.json();
+      if (response.ok) {
+        console.log(response)
+        return console.log('My book requests', jsonData);
+        //placeholder until results page
+      }
```

Comment:
If the response is not ok, you want to display an error message. Since you have a try/catch, the easiest thing to do is to throw if `!response.ok`.